installer/rpm/algorand/algorand-centos8-stream.repo

```diff
@@ -0,0 +1,6 @@
+[algorand]
+name=Algorand-centos8-stream
+baseurl=https://releases.algorand.com/rpm/stable/
```

Comment:
Can this co-exist with the centos 7 repository?

Reply:
The name is different, so should be ok.

Reply:
@onetechnical I created a private repo with Gemfury, uploaded the centos7 and centos stream 8 packages, and verified that I could install the respective algorand packages for the centos7 and centos stream 8 operating systems.  This works ok since they have unique names.  See the image below:

![image](https://user-images.githubusercontent.com/9044640/123308591-79bf9800-d4e9-11eb-81dc-ed7e7f65fdd8.png)

Reply:
Updated so that there is a single package for both centos 7 & 8, which uses dnf-automatic.